ci/jest-reporter.js

```diff
@@ -56,29 +105,40 @@ function buildDiscordStart(results, runId) {
           name: 'Started at',
           value: startedAt
         },
+        {
+          name: 'Log file (Currently all test run tasks logs running shown, will fix to one relevant one in future)',
```

Comment:
Not necessary to include this note in the actual discord message. It is sufficient to just have it display as 'Logs' and the links
```suggestion
          name: 'Logs',
```

Reply:
Changed